### DISCUSSION TITLE
Smartdevice link<title>

### DISCUSSION THREAD

**Discussion by jimburm** [community/community#47160] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

What is smart device link. Showed up on my phone and messed up my connection to Lincoln way app. Link on phone takes me to github?

**Comment by OwaisAzad** [comment]
SmartDeviceLink (SDL) is an open-source platform that allows app developers to connect smartphone apps to in-dash infotainment systems in vehicles.

It's possible that installing SDL on your phone may have caused issues with your connection to the Lincoln Way app. To resolve this, you may try the following steps:

>Uninstall SDL: Go to your phone's settings, find the SDL app, and uninstall it. This will remove the app from your phone and should restore the connection to your Lincoln Way app.
>Restart your phone
>Check your app permissions
>Update the Lincoln Way app

**Comment by jimburm** [comment]
I do not find it listed in apps. Notification pops up when I start car that SDL is running. Can remove notification only??   On Sunday, February 12, 2023 at 02:29:34 PM CST, Owais Azad ***@***.***> wrote:  
 
 


SmartDeviceLink (SDL) is an open-source platform that allows app developers to connect smartphone apps to in-dash infotainment systems in vehicles.

It's possible that installing SDL on your phone may have caused issues with your connection to the Lincoln Way app. To resolve this, you may try the following steps:


Uninstall SDL: Go to your phone's settings, find the SDL app, and uninstall it. This will remove the app from your phone and should restore the connection to your Lincoln Way app.
Restart your phone
Check your app permissions
Update the Lincoln Way app
